terraform/provision/eks-provision.tf

```diff
@@ -88,7 +92,7 @@ module "vpc" {
 
   aws_region           = local.region
   az_count             = 2
-  aws_azs              = "us-east-1b, us-east-1c"
+  aws_azs              = "${var.region}b, ${var.region}c"
```

Comment:
This should probably use the [az data source](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/data-sources/availability_zones), but we can worry about that when we make the az_count configurable.